src/fileActions.jsx

```diff
@@ -474,10 +474,19 @@ export const fileActions = (path, selected, setSelected, clipboard, setClipboard
     const spawnPaste = (sourcePath, targetPath) => {
         cockpit.spawn([
             "cp",
-            "-R",
+            "--archive",
+            "--recursive",
+            "--interactive",
             ...sourcePath,
             targetPath
-        ]).catch(err => addAlert(err.message, "danger", new Date().getTime()));
+        ], { superuser: "try", environ: ["LC_ALL=C"] }).input()
+                .catch(err => {
+                    if (err.message.match(/cp: overwrite .*?/)) {
+                        addAlert(_("Paste failed, not overwriting files"), "danger", new Date().getTime());
```

Comment:
Hmm, that's quite hackish, and the error message doesn't contain the conflicting file, so for large directories it may be hard to see what's wrong? If you parse error messages anyway, perhaps capture  the `.*` in group and show it as "not overwriting $0"?

Not a blocker, but this feels like "we really should fix this" material.

Reply:
This no longer exists? I am not sure where you found it?

Reply:
Sorry, my first review round only applied to the first commit, and I forgot to review the other commits. But let's sort out the central superuser question first.